factory/consensus/consensusComponents.go

```diff
@@ -30,14 +33,14 @@ import (
 	"github.com/multiversx/mx-chain-go/state/syncer"
 	"github.com/multiversx/mx-chain-go/trie/statistics"
 	"github.com/multiversx/mx-chain-go/update"
-	logger "github.com/multiversx/mx-chain-logger-go"
-	"github.com/multiversx/mx-chain-storage-go/timecache"
 )
 
 var log = logger.GetOrCreate("factory")
 
 const defaultSpan = 300 * time.Second
 
+const numSignatureGoRoutinesThrottler = 64
```

Comment:
this I think can be removed, as you would have access to the same constant used for signing goroutines throttler